tests/MongoDB.Driver.TestHelpers/DriverTestConfiguration.cs

```diff
@@ -120,7 +120,7 @@ public static DisposableMongoClient CreateDisposableClient(Action<ClusterBuilder
                 useMultipleShardRouters = false;
             }
 
-            var connectionString = useMultipleShardRouters 
+            var connectionString = useMultipleShardRouters
```

Comment:
This file contains only formatting changes. I would say it's better to avoid adding this change in git history

Reply:
Reverting